charts/spinnaker/templates/_helpers.tpl

```diff
@@ -81,3 +81,11 @@ Create name of kubeconfig file to use when setting up kubernetes provider
 {{- printf "/opt/kube/%s" .Values.kubeConfig.secretKey  | toString -}}
 {{- end }}
 {{- end }}
+
+{{/*
+Create global Docker registry configuration validation skip template to
```

Comment:
This should be updated to reflect the variable naming change, right?

Reply:
Done!